scss/components/popover.scss

```diff
@@ -132,20 +132,6 @@ $block: #{$fd-namespace}-popover;
           &::after {
               display: none;
           }
-
-          &.fd-popover__body--left {
-            left: 0;
-            @include fd-rtl() {
-              right: 0;
-            }
-          }
-
-          &.fd-popover__body--right {
-            right: 0;
-            @include fd-rtl() {
-              left: 0;
-            }
-          }
```

Comment:
This block is interesting.  It seems like an override within the same file.  Up a few lines from this are `&--left` and `&--right` blocks that are also doing this, but with slightly different values.  Hmmm. 🤷‍♂ The blocks above are using a `-3px` value which makes no sense.  If we are removing these from popover, we should look into removing the other blocks as well.

Reply:
Yeah I've been slowly trying to identify needed for popover styles and "needed for fundamental-styles popover styles and moving them as I encounter them. I believe this had to do with a problem in the original pr where the popover was slightly misaligned on the emphasized button due to adding that margin for the white gap.